test/test_run.py

```diff
@@ -0,0 +1,40 @@
+# standard libraries
+import logging, os
+from unittest.mock import patch
+
+# third-party libraries
+from django.core.management import call_command
+from django.test import TestCase
+
+# local libraries
+from constants import ROOT_DIR
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class RunCommandTestCase(TestCase):
+
+    def test_handle_with_properly_configured_api_util(self):
```

Comment:
The use of `self` here and in the test below is throwing Codacy errors, but I can't remove `self` or add a `@staticmethod` decorator without throwing `TypeError`s. I'm going to ignore these for now.